singularity-eos/CMakeLists.txt

```diff
@@ -47,12 +46,25 @@ set(EOS_SRCS
 )
 
 if (SINGULARITY_BUILD_CLOSURE)
-  list(APPEND EOS_SRCS eos/singularity_eos.cpp)
-  list(APPEND EOS_HEADERS eos/singularity_eos.hpp closure/mixed_cell_models.hpp)
+  list(APPEND EOS_SRCS 
+       eos/get_sg_eos.cpp)
+  if (SINGULARITY_USE_KOKKOS)
+    list(APPEND EOS_SRCS 
+         eos/get_sg_eos_p_t.cpp
+         eos/get_sg_eos_rho_t.cpp
+         eos/get_sg_eos_rho_p.cpp
+         eos/get_sg_eos_rho_e.cpp)
+  endif()
```

Comment:
is this the right logic here? I feel like this stuff should be enabled if `BUILD_FORTRAN`? Also if some of it is rage-only, we could hide that behind an additional option?

Reply:
ok. If it's easy we could add that now. That would be a win. But if you'd rather not it can wait.

Reply:
Take a look again @Yurlungur and see if this logic is more in line with what you are wanting.

Reply:
@Yurlungur can you look at this and see if it is satisfactory? This is currently a blocker to merging this PR.

Reply:
Sorry somehow missed your previous comment. This is fine for now. I still worry that the fortran-backend stuff is more coupled than I'd like but we can change it later.